### ISSUE TITLE
[BUG]: Breadcrumbs - SelectedKey property missing

### ISSUE THREAD

**Issue by JamesSnaps** [microsoft/powercat-creator-kit#263]
**Describe the bug**
The guidance for Breadcrumbs indicates that the property 'selectedkey' should be used to identify the current selected item. This property does not appear in the property list.

**To Reproduce**
Look at the property list for the breadcrumb component.

**Expected behavior**
Selectedkey property should exist so the 'bold' item can be changed.

**Screenshots**
<img width="358" alt="Screenshot 2023-01-10 at 09 48 03" src="https://user-images.githubusercontent.com/72602305/211517832-37596d13-650e-4caf-8208-50349e0c70b4.png">
<img width="906" alt="Screenshot 2023-01-10 at 09 47 56" src="https://user-images.githubusercontent.com/72602305/211517835-fad79458-7ed0-4fd3-b20b-cfec684696d4.png">

**Comment by Ramakrishnan24689** [comment]
Hi @JamesSnaps,

Thanks for reaching out in this regard.

SelectedKey was not exposed as a separate property, so that, the approach of getting selected item key is consistent with other components like Command bar/Context menu.

In order to get the Selected key, OnSelect event of the component can be utilized with the below command.

`Self.Selected.ItemKey`.

**Comment by JamesSnaps** [comment]
Thanks for the reply @Ramakrishnan24689 - not sure if this is the same thing though. I'm referring to the default selectedkey as the bold state of a clicked item does not change, so it's always the final item that shows as highlighted.
<img width="666" alt="Screenshot 2023-01-12 at 15 09 31" src="https://user-images.githubusercontent.com/72602305/212103362-76f23ce5-153e-45bf-9c1d-e5ac8448fc8e.png">
No matter what you click in this example in the creator kit, the final item is always bold. I assume this isn't intended behaviour as it isn't then clear which option you are currently on? In other components, such as 'pivot', you can define the default 'selectedkey' and then it updates as you click. 
<img width="306" alt="Screenshot 2023-01-12 at 15 06 35" src="https://user-images.githubusercontent.com/72602305/212103760-a8972e97-acb3-4487-96f8-0d1a7e7923c1.png">

**Comment by Ramakrishnan24689** [comment]
Final item being bold irrespective of which item is selected is the intent behavior. Selected item is also highlighted (Not as bold as final item), if observed carefully. In the below screenshot, Item - '2. Sports' is highlighted as well.

<img width="875" alt="image" src="https://user-images.githubusercontent.com/11135056/212109509-385b488e-6dc4-41b4-ab84-77c06f61e44f.png">

**Comment by denisem-msft** [comment]
Correct, Breadcrumb's structure is designed to display the current location of the user, so the last item will always be bolded. See Fluent UI documentation for reference: https://developer.microsoft.com/en-us/fluentui#/controls/web/breadcrumb

You should use the breadcrumb control as complementary navigation aid for users to view current position in a directory. Were you trying to display a current position in a process?

We will update documentation to reflect that this is only an output property.
